apps/backend/supabase/templates/email_change.html

```diff
@@ -111,16 +111,21 @@ <h1 style="font-size: 16px; color: #000000; line-height: 1.5">
 										word-break: break-word;
 										font-family: &quot;Courier New&quot;, Courier, monospace;
 										border-radius: 4px;
-										mso-hide: all;
 									"
 								>
+									<p>{{ .Token }}</p>
```

Comment:
think this could be larger and it should probably come before the clickable button UX-wise